ui/dashboards/src/context/DashboardProvider/dashboard-provider-api.ts

```diff
@@ -0,0 +1,23 @@
+// Copyright 2022 The Perses Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import { useDashboardStore } from './DashboardProvider';
+
+export function useLayouts() {
```

Comment:
are we also going to move `useDashboardApp`, `usePanels` , and others to this file as well

Reply:
Yeah, good question. I was thinking we definitely could do that. To your point in chat earlier today, I wonder if there's value in just "embracing the store" and just exposing the store/selectors directly rather than trying to "hide it" a bit behind these hooks.

Reply:
I see, so exporting just `useDashboardStore`?

Reply:
i like that, more simple and less code. It gives more flexibility to developers using Dashboard :) 

are you going to do it in this PR?

Reply:
I'm not gonna do it now in this PR, but we should definitely discuss. Maybe talk with the other UI devs about pros/cons.